
<!DOCTYPE html>
<meta charset="utf-8">
<html lang="en">
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Teaser - Rapid Personalized Benchmarks for NGS Read Mapping</title>
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/bootstrap/css/bootstrap.min.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/DataTables-1.10.8/media/css/jquery.dataTables.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/c3/c3.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/extensions/bootstrap-select.min.css">
    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
    <style>
body {
  padding-top: 60px;
  position: relative;
}

.sub-header {
  padding-bottom: 10px;
  border-bottom: 1px solid #eee;
}


.sidebar {
  display: none;
}

@media (min-width: 768px) {
  .sidebar {
    position: fixed;
    top: 51px;
    bottom: 0;
    left: 0;
    z-index: 1000;
    display: block;
    padding: 20px;
    overflow-x: hidden;
    overflow-y: auto; /* Scrollable contents if viewport is shorter than content. */
    background-color: #f5f5f5;
    border-right: 1px solid #eee;
  }
}

.nav-sidebar {
  margin-right: -21px;
  margin-bottom: 20px;
  margin-left: -20px;
}
.nav-sidebar > li > a {
  padding-right: 20px;
  padding-left: 20px;
}

.nav-sidebar > .active > a,
.nav-sidebar > .active > a:hover,
.nav-sidebar > .active > a:focus {
  color: #fff;
  background-color: #428bca;
}

a.separator
{
color:#777777;
}

a.separator:hover
{
color:#777777;
}

.main {
  padding: 20px;
}
@media (min-width: 768px) {
  .main {
    padding-right: 40px;
    padding-left: 40px;
  }
}
.main .page-header {
  margin-top: 0;
}

.placeholders {
  margin-bottom: 30px;
  text-align: center;
}
.placeholders h4 {
  margin-bottom: 0;
}
.placeholder {
  margin-bottom: 20px;
}
.placeholder img {
  display: inline-block;
  border-radius: 50%;
}

section {
    padding-top: 60px;
    margin-top: -60px;
}

.bar {
  fill: steelblue;
}

.bar:hover {
  fill: brown;
}

.axis {
  font: 10px sans-serif;
}

.axis path,
.axis line {
  fill: none;
  stroke: #000;
  shape-rendering: crispEdges;
}

.x.axis path {
  display: none;
}

.main-container
{
   margin: 0 auto;
   max-width: 1300px;
}


    </style>
  </head>

<body data-spy="scroll" data-target="#sidebar">
    <nav class="navbar navbar-inverse navbar-fixed-top">
      <div class="container-fluid">
        <div class="navbar-header">
          <a class="navbar-brand" href="/">Teaser</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav">
           <li><a href="start.html">Benchmark</a></li><li><a href="#" class="separator"><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></a></li><li class="dropdown"><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Sim ONT<span class="caret"></span></a><ul class="dropdown-menu" role="menu"><li><a href="e_coli_real_ont.html">Real ONT</a></li><li><a href="e_coli_sim_ont.html">Sim ONT</a></li></ul></li><li><a href="#" class="separator"><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></a></li><li class="dropdown"><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">GraphMap2<span class="caret"></span></a><ul class="dropdown-menu" role="menu"><li><a href="e_coli_sim_ont_lra.html">LRA (def)</a></li><li><a href="e_coli_sim_ont_ngmlr.html">NGMLR (def)</a></li><li><a href="e_coli_sim_ont_minimap2_01.html">Minimap2 (p1)</a></li><li><a href="e_coli_sim_ont_lra_01.html">LRA (p1)</a></li><li><a href="e_coli_sim_ont_winnowmap_01.html">Winnowmap2 (p1)</a></li><li><a href="e_coli_sim_ont_winnowmap.html">Winnowmap2 (def)</a></li><li><a href="e_coli_sim_ont_lordfast.html">lordFAST</a></li><li><a href="e_coli_sim_ont_minimap2.html">Minimap2 (def)</a></li><li><a href="e_coli_sim_ont_graphmap.html">GraphMap2</a></li><li><a href="e_coli_sim_ont_ngmlr_01.html">NGMLR (p1)</a></li></ul></li>
          </ul>
          <div class="nav navbar-nav navbar-right" style="padding:5px;">
           
          </div>
        </div>
      </div>
    </nav>

    <div class="container-fluid main-container">
      <div class="row">
        
        <div class="col-sm-3 col-md-2 sidebar">
          <div id="sidebar">
          <ul class="nav nav-sidebar">
            <li class="active"><a href="#section1">Mapping Quality Thresholds</a></li><li><a href="#section2">Basic Statistics</a></li><li><a href="#section3">Read Failure Statistics</a></li><li><a href="#section4">Advanced Statistics</a></li><li><a href="#section5">Timing</a></li><li><a href="#section6">Additional Information</a></li><li><a href="#section7">Errors and Warnings</a></li><li><a href="#section8">Subprocess Log</a></li>
          </ul>
          
          </div>
        </div>
        <div class="col-sm-9  col-sm-offset-3 col-md-offset-2  col-md-10 main" id="content">
            <section id="section1"><div class="panel panel-default"><div class="panel-heading">Mapping Quality Thresholds</div><div class="panel-body"><p>The plot below shows the percentages of correctly and wrongly mapped reads for all mapping quality thresholds for this mapper. The values at threshold 0 therefore correspond to the unfiltered results.</p><div id="plot_mapq_thresh"></div><div align="right" style="margin-top:5px;"><div class="dropdown">
  <button class="btn btn-default btn-xs dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
    <span class="glyphicon glyphicon-download-alt" aria-hidden="true"> Export
    <span class="caret"></span>
  </button>
  <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu1">
    <li><a href="javascript:exportSVG('plot_mapq_thresh');">Plot (.svg)</a></li>
    <li><a href="e_coli_sim_ont_graphmap_mapqs.csv">Raw Data  (.csv)</a></li>
  </ul>
</div></div></div></div></section><section id="section2"><div class="panel panel-default"><div class="panel-heading">Basic Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Correctly Mapped</th><td class="col-md-2">10567</td><td class="col-md-2">99.858%</td></tr><tr><th>Wrongly Mapped</th><td>5</td><td>0.047%</td></tr><tr><th>Not Mapped</th><td>10</td><td>0.095%</td></tr><tr><th>Total</th><td>10582</td><td>100.0%</td></tbody></table></div></div></div></section><section id="section3"><div class="panel panel-default"><div class="panel-heading">Read Failure Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Not mapped</th><td class="col-md-2">10</td><td class="col-md-2">0.095%</td></tr><tr><th>Missing in mapper output</th><td>0</td><td>0.0%</td></tr><tr><th>Mapped to wrong chromosome</th><td>0</td><td>0.0%</td></tr><tr><th>Mapped to wrong position</th><td>5</td><td>0.047%</td></tr><tr><th>Mapped to wrong strand</th><td>0</td><td>0.0%</td></tbody></table></div></div></div></section><section id="section4"><div class="panel panel-default"><div class="panel-heading">Advanced Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th>Missing in comparison alignment</th><td>0</td></tr><tr><th class="col-md-6">Secondary Alignments</th><td class="col-md-4">0</td></tr><tr><th><abbr title="Harmonic Mean of Precision and Recall">F-Measure</abbr></th><td>0.999300</td></tr><tr><th><abbr title="Correctly Mapped / (Correctly Mapped + Wrongly Mapped)">Precision</abbr></th><td>0.999500</td></tr><tr><th><abbr title="Correctly Mapped / (Correctly Mapped + Unmapped)">Recall</abbr></th><td>0.999100</td></tbody></table></div></div></div></section><section id="section5"><div class="panel panel-default"><div class="panel-heading">Timing</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Raw Mapping Time</th><td class="col-md-4">89.509s</td></tr><tr><th>Effective Mapping Time</th><td>89.172s</td></tr><tr><th>Effective Init Time</th><td>0.337s</td></tr><tr><th>Effective Time Measure</th><td>Wall clock</td></tr><tr><th>Mapping Time (Wall)</th><td>89.509s</td></tr><tr><th>Mapping Time (CPU)</th><td>693.667s</td></tr><tr><th>Mapping Time (CPU User)</th><td>600.000s</td></tr><tr><th>Mapping Time (CPU System)</th><td>93.667s</td></tr><tr><th>Init Time (Wall)</th><td>0.337s</td></tr><tr><th>Init Time (CPU)</th><td>0.355s</td></tr><tr><th>Init Time (CPU User)</th><td>0.189s</td></tr><tr><th>Init Time (CPU System)</th><td>0.166s</td></tr></tbody></table></div></div></div></section><section id="section6"><div class="panel panel-default"><div class="panel-heading">Additional Information</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Mapper Memory Usage</th><td class="col-md-4">614 MB</td></tr><tr><th>Total Test Runtime (Wall)</th><td>94.583s</td></tr><tr><th>Mapper Command Line:</th><td>&nbsp;</td></tr><tr><td colspan=2>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads.fastq -o out_graphmap.sam</td></tr></tbody></table></div></div></div></section><section id="section7"><div class="panel panel-default"><div class="panel-heading">Errors and Warnings</div><div class="panel-body"><i>No problems were encountered.</i></div></div></section><section id="section8"><div class="panel panel-default"><div class="panel-heading">Subprocess Log</div><div class="panel-body"><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -I /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -I /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse111" class="collapsed">stderr</a></h4></div><div id="collapse111" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse112" class="collapsed">stdout</a></h4></div><div id="collapse112" class="panel-collapse collapse"><div class="panel-body"><pre>ERROR: Unknown parameter value: '/home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta'. Exiting.<br>
<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>0.00276494026184</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_ont</small></td></tr></tbody></table></div></div></div><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads_base.fastq -o out_graphmap.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads_base.fastq -o out_graphmap.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>378844000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse113" class="collapsed">stderr</a></h4></div><div id="collapse113" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse114" class="collapsed">stdout</a></h4></div><div id="collapse114" class="panel-collapse collapse"><div class="panel-body"><pre>[13:09:21 BuildIndexes] Loading reference sequences.<br>
[13:09:21 SetupIndex_] Loading index from file: '/home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx'.<br>
[13:09:21 Index] Memory consumption: [currentRSS = 269 MB, peakRSS = 369 MB]<br>
[13:09:21 Run] Hits will be thresholded at the percentil value (percentil: 99.000000%, frequency: 6).<br>
[13:09:21 Run] Minimizers will be used. Minimizer window length: 5<br>
[13:09:21 Run] Automatically setting the maximum allowed number of regions: max. 500, attempt to reduce after 0<br>
[13:09:21 Run] Reference genome is assumed to be linear.<br>
[13:09:21 Run] Only one alignment will be reported per mapped read.<br>
[13:09:21 ProcessReads] Reads will be loaded in batches of up to 1024 MB in size.<br>
[13:09:21 ProcessReads] Batch of 1 reads (0 MiB) loaded in 0.00 sec. (94924255435968 bases)<br>
[13:09:21 ProcessReads] Memory consumption: [currentRSS = 269 MB, peakRSS = 369 MB]<br>
[13:09:21 ProcessReads] Using 8 threads.[13:09:21 ProcessReads] [CPU time: 0.00 sec, RSS: 269 MB] Read: 0/1 (0.00%) [m: 0, u: 0], length = 60, qname: read1                                                [13:09:21 ProcessReads] [CPU time: 0.00 sec, RSS: 269 MB] Read: 1/1 (100.00%) [m: 0, u: 1]                                                                         <br>
<br>
[13:09:21 ProcessReads] Memory consumption: [currentRSS = 269 MB, peakRSS = 369 MB]<br>
<br>
[13:09:21 ProcessReads] All reads processed in 0.00 sec (or 0.00 CPU min).<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>0.173006</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>0.332451105118</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>0.175613</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_ont</small></td></tr></tbody></table></div></div></div><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads_base.fastq -o out_graphmap.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads_base.fastq -o out_graphmap.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>378972000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse115" class="collapsed">stderr</a></h4></div><div id="collapse115" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse116" class="collapsed">stdout</a></h4></div><div id="collapse116" class="panel-collapse collapse"><div class="panel-body"><pre>[13:09:22 BuildIndexes] Loading reference sequences.<br>
[13:09:22 SetupIndex_] Loading index from file: '/home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx'.<br>
[13:09:22 Index] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
[13:09:22 Run] Hits will be thresholded at the percentil value (percentil: 99.000000%, frequency: 6).<br>
[13:09:22 Run] Minimizers will be used. Minimizer window length: 5<br>
[13:09:22 Run] Automatically setting the maximum allowed number of regions: max. 500, attempt to reduce after 0<br>
[13:09:22 Run] Reference genome is assumed to be linear.<br>
[13:09:22 Run] Only one alignment will be reported per mapped read.<br>
[13:09:22 ProcessReads] Reads will be loaded in batches of up to 1024 MB in size.<br>
[13:09:22 ProcessReads] Batch of 1 reads (0 MiB) loaded in 0.00 sec. (93899991211200 bases)<br>
[13:09:22 ProcessReads] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
[13:09:22 ProcessReads] Using 8 threads.[13:09:22 ProcessReads] [CPU time: 0.00 sec, RSS: 270 MB] Read: 1/1 (100.00%) [m: 0, u: 1]                                                                         <br>
<br>
[13:09:22 ProcessReads] Memory consumption: [currentRSS = 270 MB, peakRSS = 370 MB]<br>
<br>
[13:09:22 ProcessReads] All reads processed in 0.00 sec (or 0.00 CPU min).<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>0.166223</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>0.337017059326</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>0.188963</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_ont</small></td></tr></tbody></table></div></div></div><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads.fastq -o out_graphmap.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads.fastq -o out_graphmap.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>614232000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse117" class="collapsed">stderr</a></h4></div><div id="collapse117" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse118" class="collapsed">stdout</a></h4></div><div id="collapse118" class="panel-collapse collapse"><div class="panel-body"><pre>[13:09:23 BuildIndexes] Loading reference sequences.<br>
[13:09:23 SetupIndex_] Loading index from file: '/home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx'.<br>
[13:09:23 Index] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
[13:09:23 Run] Hits will be thresholded at the percentil value (percentil: 99.000000%, frequency: 6).<br>
[13:09:23 Run] Minimizers will be used. Minimizer window length: 5<br>
[13:09:23 Run] Automatically setting the maximum allowed number of regions: max. 500, attempt to reduce after 0<br>
[13:09:23 Run] Reference genome is assumed to be linear.<br>
[13:09:23 Run] Only one alignment will be reported per mapped read.<br>
[13:09:23 ProcessReads] Reads will be loaded in batches of up to 1024 MB in size.<br>
[13:09:24 ProcessReads] Batch of 10582 reads (177 MiB) loaded in 0.70 sec. (140727459532128 bases)<br>
[13:09:24 ProcessReads] Memory consumption: [currentRSS = 438 MB, peakRSS = 438 MB]<br>
[13:09:24 ProcessReads] Using 8 threads.[13:09:24 ProcessReads] [CPU time: 0.70 sec, RSS: 438 MB] Read: 0/10582 (0.00%) [m: 0, u: 0], length = 1735, qname: S1_1                                           [13:09:24 ProcessReads] [CPU time: 0.75 sec, RSS: 440 MB] Read: 8/10582 (0.08%) [m: 1, u: 0], length = 6195, qname: S1_9                                           [13:09:24 ProcessReads] [CPU time: 0.96 sec, RSS: 444 MB] Read: 10/10582 (0.09%) [m: 3, u: 0], length = 4081, qname: S1_11                                         [13:09:24 ProcessReads] [CPU time: 1.08 sec, RSS: 446 MB] Read: 12/10582 (0.11%) [m: 5, u: 0], length = 18318, qname: S1_13                                        [13:09:24 ProcessReads] [CPU time: 2.02 sec, RSS: 467 MB] Read: 23/10582 (0.22%) [m: 16, u: 0], length = 10855, qname: S1_24                                       [13:09:24 ProcessReads] [CPU time: 2.60 sec, RSS: 473 MB] Read: 32/10582 (0.30%) [m: 25, u: 0], length = 1690, qname: S1_33                                        [13:09:24 ProcessReads] [CPU time: 2.65 sec, RSS: 474 MB] Read: 34/10582 (0.32%) [m: 27, u: 0], length = 8273, qname: S1_35                                        [13:09:24 ProcessReads] [CPU time: 3.00 sec, RSS: 475 MB] Read: 36/10582 (0.34%) [m: 30, u: 0], length = 6375, qname: S1_37                                        [13:09:24 ProcessReads] [CPU time: 3.23 sec, RSS: 477 MB] Read: 45/10582 (0.43%) [m: 38, u: 0], length = 11235, qname: S1_46                                       [13:09:24 ProcessReads] [CPU time: 3.84 sec, RSS: 485 MB] Read: 56/10582 (0.53%) [m: 49, u: 0], length = 2826, qname: S1_57                                        [13:09:25 ProcessReads] [CPU time: 3.93 sec, RSS: 485 MB] Read: 57/10582 (0.54%) [m: 50, u: 0], length = 4743, qname: S1_58                                        [13:09:25 ProcessReads] [CPU time: 4.10 sec, RSS: 485 MB] Read: 63/10582 (0.60%) [m: 56, u: 0], length = 3379, qname: S1_64                                        [13:09:25 ProcessReads] [CPU time: 4.19 sec, RSS: 485 MB] Read: 65/10582 (0.61%) [m: 58, u: 0], length = 4025, qname: S1_66                                        [13:09:25 ProcessReads] [CPU time: 4.35 sec, RSS: 489 MB] Read: 70/10582 (0.66%) [m: 63, u: 0], length = 8193, qname: S1_71                                        [13:09:25 ProcessReads] [CPU time: 4.70 sec, RSS: 491 MB] Read: 75/10582 (0.71%) [m: 68, u: 0], length = 26937, qname: S1_76                                       [13:09:25 ProcessReads] [CPU time: 6.83 sec, RSS: 507 MB] Read: 98/10582 (0.93%) [m: 91, u: 0], length = 10880, qname: S1_99                                       [13:09:25 ProcessReads] [CPU time: 7.40 sec, RSS: 508 MB] Read: 105/10582 (0.99%) [m: 98, u: 0], length = 1956, qname: S1_106                                      [13:09:25 ProcessReads] [CPU time: 7.47 sec, RSS: 508 MB] Read: 107/10582 (1.01%) [m: 100, u: 0], length = 4273, qname: S1_108                                     [13:09:25 ProcessReads] [CPU time: 7.63 sec, RSS: 509 MB] Read: 112/10582 (1.06%) [m: 105, u: 0], length = 27802, qname: S1_113                                    [13:09:25 ProcessReads] [CPU time: 9.76 sec, RSS: 515 MB] Read: 145/10582 (1.37%) [m: 138, u: 0], length = 1259, qname: S1_146                                     [13:09:25 ProcessReads] [CPU time: 9.80 sec, RSS: 515 MB] Read: 148/10582 (1.40%) [m: 141, u: 0], length = 13491, qname: S1_149                                    [13:09:25 ProcessReads] [CPU time: 10.46 sec, RSS: 516 MB] Read: 160/10582 (1.51%) [m: 153, u: 0], length = 2835, qname: S1_161                                    [13:09:25 ProcessReads] [CPU time: 10.58 sec, RSS: 519 MB] Read: 161/10582 (1.52%) [m: 154, u: 0], length = 6836, qname: S1_162                                    [13:09:25 ProcessReads] [CPU time: 10.81 sec, RSS: 519 MB] Read: 165/10582 (1.56%) [m: 158, u: 0], length = 21095, qname: S1_166                                   [13:09:26 ProcessReads] [CPU time: 12.31 sec, RSS: 519 MB] Read: 188/10582 (1.78%) [m: 181, u: 0], length = 2227, qname: S1_189                                    [13:09:26 ProcessReads] [CPU time: 12.39 sec, RSS: 519 MB] Read: 191/10582 (1.80%) [m: 184, u: 0], length = 263, qname: S1_192                                     [13:09:26 ProcessReads] [CPU time: 12.39 sec, RSS: 519 MB] Read: 192/10582 (1.81%) [m: 185, u: 0], length = 6980, qname: S1_193                                    [13:09:26 ProcessReads] [CPU time: 12.71 sec, RSS: 519 MB] Read: 197/10582 (1.86%) [m: 190, u: 0], length = 14537, qname: S1_198                                   [13:09:26 ProcessReads] [CPU time: 13.65 sec, RSS: 520 MB] Read: 213/10582 (2.01%) [m: 206, u: 0], length = 19608, qname: S1_214                                   [13:09:26 ProcessReads] [CPU time: 15.01 sec, RSS: 521 MB] Read: 233/10582 (2.20%) [m: 226, u: 0], length = 3555, qname: S1_234                                    [13:09:26 ProcessReads] [CPU time: 15.14 sec, RSS: 521 MB] Read: 235/10582 (2.22%) [m: 228, u: 0], length = 5844, qname: S1_236                                    [13:09:26 ProcessReads] [CPU time: 15.36 sec, RSS: 521 MB] Read: 239/10582 (2.26%) [m: 232, u: 0], length = 7444, qname: S1_240                                    [13:09:26 ProcessReads] [CPU time: 15.76 sec, RSS: 525 MB] Read: 242/10582 (2.29%) [m: 235, u: 0], length = 4830, qname: S1_243                                    [13:09:26 ProcessReads] [CPU time: 15.93 sec, RSS: 526 MB] Read: 244/10582 (2.31%) [m: 237, u: 0], length = 29361, qname: S1_245                                   [13:09:26 ProcessReads] [CPU time: 18.32 sec, RSS: 530 MB] Read: 266/10582 (2.51%) [m: 259, u: 0], length = 4489, qname: S1_267                                    [13:09:26 ProcessReads] [CPU time: 18.46 sec, RSS: 530 MB] Read: 269/10582 (2.54%) [m: 262, u: 0], length = 20839, qname: S1_270                                   [13:09:27 ProcessReads] [CPU time: 20.38 sec, RSS: 531 MB] Read: 296/10582 (2.80%) [m: 289, u: 0], length = 10165, qname: S1_297                                   [13:09:27 ProcessReads] [CPU time: 20.89 sec, RSS: 531 MB] Read: 301/10582 (2.84%) [m: 294, u: 0], length = 11325, qname: S1_302                                   [13:09:27 ProcessReads] [CPU time: 21.41 sec, RSS: 531 MB] Read: 316/10582 (2.99%) [m: 309, u: 0], length = 13861, qname: S1_317                                   [13:09:27 ProcessReads] [CPU time: 22.11 sec, RSS: 531 MB] Read: 329/10582 (3.11%) [m: 322, u: 0], length = 4725, qname: S1_330                                    [13:09:27 ProcessReads] [CPU time: 22.29 sec, RSS: 531 MB] Read: 331/10582 (3.13%) [m: 324, u: 0], length = 2126, qname: S1_332                                    [13:09:27 ProcessReads] [CPU time: 22.37 sec, RSS: 531 MB] Read: 333/10582 (3.15%) [m: 326, u: 0], length = 5716, qname: S1_334                                    [13:09:27 ProcessReads] [CPU time: 22.58 sec, RSS: 533 MB] Read: 338/10582 (3.19%) [m: 331, u: 0], length = 12872, qname: S1_339                                   [13:09:27 ProcessReads] [CPU time: 23.32 sec, RSS: 533 MB] Read: 348/10582 (3.29%) [m: 341, u: 0], length = 2853, qname: S1_349                                    [13:09:27 ProcessReads] [CPU time: 23.46 sec, RSS: 533 MB] Read: 350/10582 (3.31%) [m: 343, u: 0], length = 8001, qname: S1_351                                    [13:09:27 ProcessReads] [CPU time: 23.76 sec, RSS: 533 MB] Read: 357/10582 (3.37%) [m: 350, u: 0], length = 1721, qname: S1_358                                    [13:09:27 ProcessReads] [CPU time: 23.81 sec, RSS: 534 MB] Read: 359/10582 (3.39%) [m: 352, u: 0], length = 18029, qname: S1_360                                   [13:09:27 ProcessReads] [CPU time: 25.30 sec, RSS: 536 MB] Read: 374/10582 (3.53%) [m: 367, u: 0], length = 2133, qname: S1_375                                    [13:09:27 ProcessReads] [CPU time: 25.39 sec, RSS: 536 MB] Read: 375/10582 (3.54%) [m: 368, u: 0], length = 6724, qname: S1_376                                    [13:09:27 ProcessReads] [CPU time: 25.75 sec, RSS: 536 MB] Read: 383/10582 (3.62%) [m: 376, u: 0], length = 5541, qname: S1_384                                    [13:09:27 ProcessReads] [CPU time: 25.98 sec, RSS: 536 MB] Read: 386/10582 (3.65%) [m: 379, u: 0], length = 10989, qname: S1_387                                   [13:09:27 ProcessReads] [CPU time: 26.48 sec, RSS: 537 MB] Read: 391/10582 (3.69%) [m: 384, u: 0], length = 23833, qname: S1_392                                   [13:09:28 ProcessReads] [CPU time: 28.47 sec, RSS: 537 MB] Read: 413/10582 (3.90%) [m: 406, u: 0], length = 3549, qname: S1_414                                    [13:09:28 ProcessReads] [CPU time: 28.69 sec, RSS: 537 MB] Read: 414/10582 (3.91%) [m: 407, u: 0], length = 33751, qname: S1_415                                   [13:09:28 ProcessReads] [CPU time: 31.45 sec, RSS: 539 MB] Read: 449/10582 (4.24%) [m: 442, u: 0], length = 5252, qname: S1_450                                    [13:09:28 ProcessReads</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>93.66688</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>89.5092840195</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>599.999807</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_ont</small></td></tr></tbody></table></div></div></div></div></div></section><p></p><div style="margin:300px; height:300px;">&nbsp;</div>
        </div>
      </div>
    </div>


<script src="http://teaser.cibiv.univie.ac.at/static/js/d3.v3.min.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/c3/c3.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/js/jquery.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/js/svgexport.js"></script>
<script>

var cols=[["Correct", 0.9985824985824986, 0.9985824985824986, 0.9984879984879985, 0.9984879984879985, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], ["Wrong", -0.8321520821520821, -0.8321520821520821, -0.8321520821520821, -0.33286083286083284, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]];
var a=0;
var b=0;
var last_a=0;
var last_b=0;

function updateRange()
{
	if(last_a==a && last_b==b)
	{
		setTimeout(updateRange,500);
		return;
	}
 
        an=Math.floor(parseInt(a));
        bn=Math.floor(parseInt(b));
	var mn = 0;
	var mx = 0;
	for(i=an;i<bn;i++)
	{
		mx=Math.max(mx,cols[0][i+1]);
		mn=Math.min(mn,cols[1][i+1]);
	}
	r={}
	r.max=mx;
	r.min=mn;
	chart.axis.range(r);
	last_a=a;
	last_b=b;
	setTimeout(updateRange,500);
}
setTimeout(updateRange,500);

var chart = c3.generate({
bindto: '#plot_mapq_thresh',
data: {
  columns: [["Correct", 0.9985824985824986, 0.9985824985824986, 0.9984879984879985, 0.9984879984879985, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.9914949914949915, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], ["Wrong", -0.8321520821520821, -0.8321520821520821, -0.8321520821520821, -0.33286083286083284, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, -0.16643041643041642, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]],
  type: "area"
},
grid: {
  y: {
    show: true
  }
},
axis: {
  x: {
    label: "Mapping Quality Threshold"
  },

  y: {
    label: { text: "Read Count", position: "outer-middle" },
    tick: {
       format: function(v) { if(v>=0){return Math.min(100,(Math.round(100000*v)/1000))+"%";}else{ return Math.min(100,(Math.round(100000*(-v/1761.166667))/1000))+"%";} }
    }
  }
},
legend: {
	position: "right"
},
subchart: {
	show: true,
	onbrush: function (d) {
		a = d[0];
		b = d[1];
	}
}
});

</script>
<script src="http://teaser.cibiv.univie.ac.at/static/bootstrap/js/bootstrap.min.js"></script>
<script type="javascript">$('#content').scrollspy();</script>
<script src="http://teaser.cibiv.univie.ac.at/static/extensions/bootstrap-select.min.js"></script>

<script src="http://teaser.cibiv.univie.ac.at/static/DataTables-1.10.8/media/js/jquery.dataTables.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/extensions/dataTables.bootstrap.js"></script>
</body>
</html>
