
<!DOCTYPE html>
<meta charset="utf-8">
<html lang="en">
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Teaser - Rapid Personalized Benchmarks for NGS Read Mapping</title>
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/bootstrap/css/bootstrap.min.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/DataTables-1.10.8/media/css/jquery.dataTables.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/c3/c3.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/extensions/bootstrap-select.min.css">
    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
    <style>
body {
  padding-top: 60px;
  position: relative;
}

.sub-header {
  padding-bottom: 10px;
  border-bottom: 1px solid #eee;
}


.sidebar {
  display: none;
}

@media (min-width: 768px) {
  .sidebar {
    position: fixed;
    top: 51px;
    bottom: 0;
    left: 0;
    z-index: 1000;
    display: block;
    padding: 20px;
    overflow-x: hidden;
    overflow-y: auto; /* Scrollable contents if viewport is shorter than content. */
    background-color: #f5f5f5;
    border-right: 1px solid #eee;
  }
}

.nav-sidebar {
  margin-right: -21px;
  margin-bottom: 20px;
  margin-left: -20px;
}
.nav-sidebar > li > a {
  padding-right: 20px;
  padding-left: 20px;
}

.nav-sidebar > .active > a,
.nav-sidebar > .active > a:hover,
.nav-sidebar > .active > a:focus {
  color: #fff;
  background-color: #428bca;
}

a.separator
{
color:#777777;
}

a.separator:hover
{
color:#777777;
}

.main {
  padding: 20px;
}
@media (min-width: 768px) {
  .main {
    padding-right: 40px;
    padding-left: 40px;
  }
}
.main .page-header {
  margin-top: 0;
}

.placeholders {
  margin-bottom: 30px;
  text-align: center;
}
.placeholders h4 {
  margin-bottom: 0;
}
.placeholder {
  margin-bottom: 20px;
}
.placeholder img {
  display: inline-block;
  border-radius: 50%;
}

section {
    padding-top: 60px;
    margin-top: -60px;
}

.bar {
  fill: steelblue;
}

.bar:hover {
  fill: brown;
}

.axis {
  font: 10px sans-serif;
}

.axis path,
.axis line {
  fill: none;
  stroke: #000;
  shape-rendering: crispEdges;
}

.x.axis path {
  display: none;
}

.main-container
{
   margin: 0 auto;
   max-width: 1300px;
}


    </style>
  </head>

<body data-spy="scroll" data-target="#sidebar">
    <nav class="navbar navbar-inverse navbar-fixed-top">
      <div class="container-fluid">
        <div class="navbar-header">
          <a class="navbar-brand" href="/">Teaser</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav">
           <li><a href="start.html">Benchmark</a></li><li><a href="#" class="separator"><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></a></li><li><a href="e_coli_sim_clr.html">Sim CLR</a></li><li><a href="#" class="separator"><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></a></li><li class="dropdown"><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">NGMLR (p1)<span class="caret"></span></a><ul class="dropdown-menu" role="menu"><li><a href="e_coli_sim_clr_lra.html">LRA (def)</a></li><li><a href="e_coli_sim_clr_ngmlr.html">NGMLR (def)</a></li><li><a href="e_coli_sim_clr_minimap2_01.html">Minimap2 (p1)</a></li><li><a href="e_coli_sim_clr_lra_01.html">LRA (p1)</a></li><li><a href="e_coli_sim_clr_winnowmap_01.html">Winnowmap2 (p1)</a></li><li><a href="e_coli_sim_clr_winnowmap.html">Winnowmap2 (def)</a></li><li><a href="e_coli_sim_clr_lordfast.html">lordFAST</a></li><li><a href="e_coli_sim_clr_minimap2.html">Minimap2 (def)</a></li><li><a href="e_coli_sim_clr_graphmap.html">GraphMap2</a></li><li><a href="e_coli_sim_clr_ngmlr_01.html">NGMLR (p1)</a></li></ul></li>
          </ul>
          <div class="nav navbar-nav navbar-right" style="padding:5px;">
           
          </div>
        </div>
      </div>
    </nav>

    <div class="container-fluid main-container">
      <div class="row">
        
        <div class="col-sm-3 col-md-2 sidebar">
          <div id="sidebar">
          <ul class="nav nav-sidebar">
            <li class="active"><a href="#section1">Mapping Quality Thresholds</a></li><li><a href="#section2">Basic Statistics</a></li><li><a href="#section3">Read Failure Statistics</a></li><li><a href="#section4">Advanced Statistics</a></li><li><a href="#section5">Timing</a></li><li><a href="#section6">Additional Information</a></li><li><a href="#section7">Errors and Warnings</a></li><li><a href="#section8">Subprocess Log</a></li>
          </ul>
          
          </div>
        </div>
        <div class="col-sm-9  col-sm-offset-3 col-md-offset-2  col-md-10 main" id="content">
            <section id="section1"><div class="panel panel-default"><div class="panel-heading">Mapping Quality Thresholds</div><div class="panel-body"><p>The plot below shows the percentages of correctly and wrongly mapped reads for all mapping quality thresholds for this mapper. The values at threshold 0 therefore correspond to the unfiltered results.</p><div id="plot_mapq_thresh"></div><div align="right" style="margin-top:5px;"><div class="dropdown">
  <button class="btn btn-default btn-xs dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
    <span class="glyphicon glyphicon-download-alt" aria-hidden="true"> Export
    <span class="caret"></span>
  </button>
  <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu1">
    <li><a href="javascript:exportSVG('plot_mapq_thresh');">Plot (.svg)</a></li>
    <li><a href="e_coli_sim_clr_ngmlr_01_mapqs.csv">Raw Data  (.csv)</a></li>
  </ul>
</div></div></div></div></section><section id="section2"><div class="panel panel-default"><div class="panel-heading">Basic Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Correctly Mapped</th><td class="col-md-2">9709</td><td class="col-md-2">96.732%</td></tr><tr><th>Wrongly Mapped</th><td>256</td><td>2.551%</td></tr><tr><th>Not Mapped</th><td>72</td><td>0.717%</td></tr><tr><th>Total</th><td>10037</td><td>100.0%</td></tbody></table></div></div></div></section><section id="section3"><div class="panel panel-default"><div class="panel-heading">Read Failure Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Not mapped</th><td class="col-md-2">66</td><td class="col-md-2">0.658%</td></tr><tr><th>Missing in mapper output</th><td>6</td><td>0.06%</td></tr><tr><th>Mapped to wrong chromosome</th><td>0</td><td>0.0%</td></tr><tr><th>Mapped to wrong position</th><td>256</td><td>2.551%</td></tr><tr><th>Mapped to wrong strand</th><td>0</td><td>0.0%</td></tbody></table></div></div></div></section><section id="section4"><div class="panel panel-default"><div class="panel-heading">Advanced Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th>Missing in comparison alignment</th><td>6</td></tr><tr><th class="col-md-6">Secondary Alignments</th><td class="col-md-4">138</td></tr><tr><th><abbr title="Harmonic Mean of Precision and Recall">F-Measure</abbr></th><td>0.983700</td></tr><tr><th><abbr title="Correctly Mapped / (Correctly Mapped + Wrongly Mapped)">Precision</abbr></th><td>0.974300</td></tr><tr><th><abbr title="Correctly Mapped / (Correctly Mapped + Unmapped)">Recall</abbr></th><td>0.993200</td></tbody></table></div></div></div></section><section id="section5"><div class="panel panel-default"><div class="panel-heading">Timing</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Raw Mapping Time</th><td class="col-md-4">136.271s</td></tr><tr><th>Effective Mapping Time</th><td>133.959s</td></tr><tr><th>Effective Init Time</th><td>2.313s</td></tr><tr><th>Effective Time Measure</th><td>Wall clock</td></tr><tr><th>Mapping Time (Wall)</th><td>136.271s</td></tr><tr><th>Mapping Time (CPU)</th><td>1010.146s</td></tr><tr><th>Mapping Time (CPU User)</th><td>1004.405s</td></tr><tr><th>Mapping Time (CPU System)</th><td>5.741s</td></tr><tr><th>Init Time (Wall)</th><td>2.313s</td></tr><tr><th>Init Time (CPU)</th><td>2.547s</td></tr><tr><th>Init Time (CPU User)</th><td>0.308s</td></tr><tr><th>Init Time (CPU System)</th><td>2.239s</td></tr></tbody></table></div></div></div></section><section id="section6"><div class="panel panel-default"><div class="panel-heading">Additional Information</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Mapper Memory Usage</th><td class="col-md-4">3913 MB</td></tr><tr><th>Total Test Runtime (Wall)</th><td>146.135s</td></tr><tr><th>Mapper Command Line:</th><td>&nbsp;</td></tr><tr><td colspan=2>/home/raven/mambaforge/envs/dipenv/bin/ngmlr  -x pacbio -t 8 -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -q reads.fastq -o out_ngmlr_01.sam</td></tr></tbody></table></div></div></div></section><section id="section7"><div class="panel panel-default"><div class="panel-heading">Errors and Warnings</div><div class="panel-body"><i>No problems were encountered.</i></div></div></section><section id="section8"><div class="panel panel-default"><div class="panel-heading">Subprocess Log</div><div class="panel-body"><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/mambaforge/envs/dipenv/bin/ngmlr  -x pacbio -t 8 -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -q reads_base.fastq -o out_ngmlr_01.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/mambaforge/envs/dipenv/bin/ngmlr  -x pacbio -t 8 -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -q reads_base.fastq -o out_ngmlr_01.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>2439780000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse57" class="collapsed">stderr</a></h4></div><div id="collapse57" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse58" class="collapsed">stdout</a></h4></div><div id="collapse58" class="panel-collapse collapse"><div class="panel-body"><pre>ngmlr 0.2.7 (build: May 16 2023 08:30:10, start: 2023-05-31.14:07:33)<br>
Contact: philipp.rescheneder@univie.ac.at<br>
Opening for output (SAM): out_ngmlr_01.sam<br>
Reading encoded reference from /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta-enc.2.ngm<br>
Reading 4 Mbp from disk took 0.00s<br>
Reading reference index from /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta-ht-13-2.2.ngm<br>
Reading from disk took 0.47s<br>
Opening query file reads_base.fastq<br>
Mapping reads...<br>
Processed: 1 (0.00), R/S: 0.50, RL: 60, Time: 50.00 0.00 0.00, Align: 1.00, 158, 0.00<br>
[A[2KProcessed: 1 (0.00), R/S: 0.50, RL: 60, Time: 50.00 0.00 0.00, Align: 1.00, 158, 0.00<br>
Done (0 reads mapped (0.00%), 1 reads not mapped, 1 lines written)(elapsed: 0m, 0 r/s)<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>2.428038</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>2.50185203552</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>0.365129</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_clr</small></td></tr></tbody></table></div></div></div><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/mambaforge/envs/dipenv/bin/ngmlr  -x pacbio -t 8 -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -q reads_base.fastq -o out_ngmlr_01.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/mambaforge/envs/dipenv/bin/ngmlr  -x pacbio -t 8 -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -q reads_base.fastq -o out_ngmlr_01.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>2287984000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse59" class="collapsed">stderr</a></h4></div><div id="collapse59" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse60" class="collapsed">stdout</a></h4></div><div id="collapse60" class="panel-collapse collapse"><div class="panel-body"><pre>ngmlr 0.2.7 (build: May 16 2023 08:30:10, start: 2023-05-31.14:07:36)<br>
Contact: philipp.rescheneder@univie.ac.at<br>
Opening for output (SAM): out_ngmlr_01.sam<br>
Reading encoded reference from /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta-enc.2.ngm<br>
Reading 4 Mbp from disk took 0.00s<br>
Reading reference index from /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta-ht-13-2.2.ngm<br>
Reading from disk took 0.28s<br>
Opening query file reads_base.fastq<br>
Mapping reads...<br>
Processed: 1 (0.00), R/S: 0.50, RL: 60, Time: 50.00 0.00 0.00, Align: 1.00, 158, 0.00<br>
[A[2KProcessed: 1 (0.00), R/S: 0.50, RL: 60, Time: 50.00 0.00 0.00, Align: 1.00, 158, 0.00<br>
Done (0 reads mapped (0.00%), 1 reads not mapped, 1 lines written)(elapsed: 0m, 0 r/s)<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>2.239313</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>2.31265211105</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>0.308121</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_clr</small></td></tr></tbody></table></div></div></div><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/mambaforge/envs/dipenv/bin/ngmlr  -x pacbio -t 8 -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -q reads.fastq -o out_ngmlr_01.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/mambaforge/envs/dipenv/bin/ngmlr  -x pacbio -t 8 -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -q reads.fastq -o out_ngmlr_01.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>3913892000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse61" class="collapsed">stderr</a></h4></div><div id="collapse61" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse62" class="collapsed">stdout</a></h4></div><div id="collapse62" class="panel-collapse collapse"><div class="panel-body"><pre>ngmlr 0.2.7 (build: May 16 2023 08:30:10, start: 2023-05-31.14:07:39)<br>
Contact: philipp.rescheneder@univie.ac.at<br>
Opening for output (SAM): out_ngmlr_01.sam<br>
Reading encoded reference from /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta-enc.2.ngm<br>
Reading 4 Mbp from disk took 0.00s<br>
Reading reference index from /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta-ht-13-2.2.ngm<br>
Reading from disk took 0.21s<br>
Opening query file reads.fastq<br>
Mapping reads...<br>
Processed: 134 (0.99), R/S: 67.00, RL: 13106, Time: 3.06 1.73 92.59, Align: 1.00, 394, 1.00<br>
[A[2KProcessed: 295 (0.99), R/S: 73.75, RL: 10435, Time: 2.79 2.01 92.84, Align: 1.00, 385, 0.99<br>
[A[2KProcessed: 473 (0.99), R/S: 78.83, RL: 9997, Time: 2.04 1.91 93.09, Align: 1.00, 380, 0.99<br>
[A[2KProcessed: 647 (0.99), R/S: 80.87, RL: 9527, Time: 2.56 2.05 92.38, Align: 1.00, 380, 0.99<br>
[A[2KProcessed: 813 (0.99), R/S: 81.30, RL: 9589, Time: 2.00 2.00 91.86, Align: 1.00, 379, 0.99<br>
[A[2KProcessed: 970 (0.99), R/S: 80.83, RL: 9545, Time: 2.00 2.00 92.65, Align: 1.00, 377, 0.99<br>
[A[2KProcessed: 1086 (0.99), R/S: 77.57, RL: 9796, Time: 2.00 2.00 92.41, Align: 1.00, 384, 0.99<br>
[A[2KProcessed: 1231 (0.99), R/S: 76.93, RL: 9607, Time: 2.00 2.00 93.15, Align: 1.00, 385, 0.99<br>
[A[2KProcessed: 1397 (0.99), R/S: 77.61, RL: 9653, Time: 2.00 2.00 93.62, Align: 1.00, 385, 0.99<br>
[A[2KProcessed: 1548 (0.99), R/S: 77.40, RL: 9466, Time: 2.00 2.00 93.10, Align: 1.00, 387, 0.99<br>
[A[2KProcessed: 1721 (0.99), R/S: 78.22, RL: 9498, Time: 2.00 2.00 93.14, Align: 1.00, 387, 0.99<br>
[A[2KProcessed: 1886 (0.99), R/S: 78.58, RL: 9407, Time: 2.00 2.00 93.53, Align: 1.00, 387, 0.99<br>
[A[2KProcessed: 2036 (0.99), R/S: 78.30, RL: 9422, Time: 2.00 2.00 93.26, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 2191 (0.99), R/S: 78.25, RL: 9349, Time: 2.00 2.00 93.50, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 2339 (0.99), R/S: 77.96, RL: 9391, Time: 2.00 2.00 93.00, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 2474 (0.99), R/S: 77.31, RL: 9430, Time: 2.00 2.00 92.87, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 2632 (0.99), R/S: 77.41, RL: 9369, Time: 2.00 1.75 91.95, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 2807 (0.99), R/S: 77.97, RL: 9392, Time: 2.00 2.00 93.07, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 2956 (0.99), R/S: 77.79, RL: 9358, Time: 2.00 2.00 93.50, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 3110 (0.99), R/S: 77.75, RL: 9353, Time: 2.00 2.00 92.64, Align: 1.00, 388, 0.99<br>
[A[2KProcessed: 3255 (0.99), R/S: 77.50, RL: 9330, Time: 2.00 2.00 92.01, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 3412 (0.99), R/S: 77.54, RL: 9349, Time: 2.00 2.00 92.79, Align: 1.00, 388, 0.99<br>
[A[2KProcessed: 3577 (0.99), R/S: 77.76, RL: 9345, Time: 2.00 2.00 92.04, Align: 1.00, 388, 0.99<br>
[A[2KProcessed: 3710 (0.99), R/S: 77.29, RL: 9323, Time: 2.00 2.00 92.00, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 3833 (0.99), R/S: 76.66, RL: 9370, Time: 2.00 2.00 93.53, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 3975 (0.99), R/S: 76.44, RL: 9354, Time: 2.00 2.00 93.58, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 4134 (0.99), R/S: 76.55, RL: 9321, Time: 2.00 2.00 93.31, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 4286 (0.99), R/S: 76.53, RL: 9347, Time: 2.00 2.00 92.64, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 4444 (0.99), R/S: 76.62, RL: 9322, Time: 2.00 2.00 93.18, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 4613 (0.99), R/S: 76.88, RL: 9303, Time: 2.00 2.00 93.21, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 4787 (0.99), R/S: 77.21, RL: 9277, Time: 2.00 2.00 93.13, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 4923 (0.99), R/S: 76.92, RL: 9283, Time: 2.00 2.00 93.09, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 5055 (0.99), R/S: 76.59, RL: 9274, Time: 2.00 2.00 92.25, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 5201 (0.99), R/S: 76.48, RL: 9262, Time: 2.00 2.00 92.36, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 5341 (0.99), R/S: 76.30, RL: 9284, Time: 2.00 2.00 93.50, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 5488 (0.99), R/S: 76.22, RL: 9289, Time: 2.00 2.00 93.87, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 5632 (0.99), R/S: 76.10, RL: 9298, Time: 2.00 2.00 93.37, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 5793 (0.99), R/S: 76.22, RL: 9314, Time: 2.00 2.00 92.94, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 5938 (0.99), R/S: 76.12, RL: 9269, Time: 2.00 2.00 93.29, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 6095 (0.99), R/S: 76.18, RL: 9271, Time: 2.00 2.00 93.30, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 6245 (0.99), R/S: 76.15, RL: 9279, Time: 2.00 2.00 92.95, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 6399 (0.99), R/S: 76.17, RL: 9311, Time: 2.00 2.00 93.01, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 6556 (0.99), R/S: 76.23, RL: 9268, Time: 2.00 2.00 92.94, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 6715 (0.99), R/S: 76.30, RL: 9262, Time: 2.00 2.00 93.47, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 6905 (0.99), R/S: 76.72, RL: 9240, Time: 2.00 2.00 93.03, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 7057 (0.99), R/S: 76.70, RL: 9241, Time: 2.00 2.00 93.03, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 7206 (0.99), R/S: 76.66, RL: 9241, Time: 2.00 2.00 92.62, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 7361 (0.99), R/S: 76.67, RL: 9262, Time: 2.00 2.00 93.03, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 7517 (0.99), R/S: 76.70, RL: 9258, Time: 2.00 2.00 92.91, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 7667 (0.99), R/S: 76.67, RL: 9265, Time: 2.00 2.00 92.91, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 7819 (0.99), R/S: 76.65, RL: 9274, Time: 2.00 2.00 93.09, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 7951 (0.99), R/S: 76.45, RL: 9267, Time: 2.00 2.00 92.52, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 8097 (0.99), R/S: 76.38, RL: 9281, Time: 2.00 2.00 93.51, Align: 1.00, 389, 0.99<br>
[A[2KProcessed: 8224 (0.99), R/S: 76.14, RL: 9285, Time: 2.00 2.00 92.97, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 8355 (0.99), R/S: 75.95, RL: 9299, Time: 2.00 2.00 93.06, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 8486 (0.99), R/S: 75.76, RL: 9285, Time: 2.00 2.00 93.27, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 8635 (0.99), R/S: 75.74, RL: 9277, Time: 2.00 2.00 93.03, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 8798 (0.99), R/S: 75.84, RL: 9251, Time: 2.00 2.00 93.01, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 8943 (0.99), R/S: 75.78, RL: 9270, Time: 2.00 2.00 93.13, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 9088 (0.99), R/S: 75.73, RL: 9296, Time: 2.00 2.00 92.99, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 9223 (0.99), R/S: 75.59, RL: 9280, Time: 2.00 2.00 92.87, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 9370 (0.99), R/S: 75.56, RL: 9301, Time: 2.00 2.00 92.50, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 9504 (0.99), R/S: 75.42, RL: 9284, Time: 2.00 2.00 93.12, Align: 1.00, 390, 0.99<br>
[A[2KProcessed: 9647 (0.99), R/S: 75.36, RL: 9286, Time: 2.00 2.00 93.25, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 9792 (0.99), R/S: 75.32, RL: 9284, Time: 2.00 2.00 93.52, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 9945 (0.99), R/S: 75.34, RL: 9284, Time: 2.00 2.00 93.06, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 10036 (0.99), R/S: 74.89, RL: 9250, Time: 2.00 2.00 93.07, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 10037 (0.99), R/S: 73.80, RL: 9249, Time: 2.00 2.00 93.03, Align: 1.00, 391, 0.99<br>
[A[2KProcessed: 10037 (0.99), R/S: 73.80, RL: 9249, Time: 2.00 2.00 93.03, Align: 1.00, 391, 0.99<br>
Done (9965 reads mapped (99.28%), 72 reads not mapped, 10175 lines written)(elapsed: 2m, 73 r/s)<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>5.740964</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>136.27127099</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>1004.404564</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_clr</small></td></tr></tbody></table></div></div></div></div></div></section><p></p><div style="margin:300px; height:300px;">&nbsp;</div>
        </div>
      </div>
    </div>


<script src="http://teaser.cibiv.univie.ac.at/static/js/d3.v3.min.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/c3/c3.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/js/jquery.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/js/svgexport.js"></script>
<script>

var cols=[["Correct", 0.9673209126232938, 0.9673209126232938, 0.9673209126232938, 0.9673209126232938, 0.9672212812593405, 0.9672212812593405, 0.9672212812593405, 0.9672212812593405, 0.9672212812593405, 0.9672212812593405, 0.9671216498953871, 0.9670220185314337, 0.9669223871674804, 0.966822755803527, 0.9665238617116668, 0.9665238617116668, 0.9662249676198067, 0.9661253362558534, 0.9658264421639933, 0.9655275480721331, 0.9654279167081797, 0.9649297598884129, 0.9647304971605061, 0.9643319717046926, 0.9642323403407392, 0.963634552157019, 0.9633356580651589, 0.9626382385174853, 0.9623393444256252, 0.9615422935139982, 0.9611437680581847, 0.9579555644116768, 0.9565607253163296, 0.955066254857029, 0.9527747334861014, 0.9506824748430805, 0.9481916907442463, 0.944903855733785, 0.9408189698116968, 0.936634452525655, 0.9326491979675202, 0.9266713161303178, 0.9217893792966025, 0.9145162897280064, 0.9080402510710371, 0.9001693733187207, 0.8874165587326891, 0.8775530537013052, 0.8676895486699213, 0.8463684367838996, 0.8263425326292717, 0.7909733984258245, 0.7563016837700508, 0.7106705190794061, 0.6585633157317924, 0.5876257845969911, 0.500647603865697, 0.3985254558134901, 0.29520773139384276, 0.22705987844973596, 0.21908936933346618, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], ["Wrong", -0.9635570180216467, -0.9522653342167055, -0.9522653342167055, -0.9522653342167055, -0.9522653342167055, -0.9522653342167055, -0.9485014396150584, -0.9409736504117643, -0.9409736504117643, -0.9409736504117643, -0.9334458612084703, -0.9296819666068231, -0.9259180720051761, -0.9259180720051761, -0.9221541774035291, -0.9221541774035291, -0.9221541774035291, -0.9221541774035291, -0.918390282801882, -0.910862493598588, -0.9070985989969408, -0.8958069151919996, -0.8845152313870585, -0.8807513367854115, -0.8732235475821173, -0.854404074573882, -0.8393484961672938, -0.8130012339557644, -0.7715983933376468, -0.7301955527195291, -0.7038482905079998, -0.6737371336948232, -0.6360981876783527, -0.6097509254668233, -0.5834036632552939, -0.5269452442305881, -0.5156535604256469, -0.4893062982141175, -0.455431246799294, -0.41402840618117626, -0.3801533547663528, -0.3425144087498822, -0.31240325193670576, -0.28981988432682343, -0.27100041131858815, -0.24465314910705874, -0.2295975707004705, -0.22206978149717638, -0.2107780976922352, -0.18066694087905874, -0.16184746787082346, -0.15431967866752935, -0.10915294344776466, -0.09409736504117643, -0.0790417866345882, -0.06775010282964704, -0.06022231362635292, -0.04140284061811764, -0.03387505141482352, -0.03387505141482352, -0.03387505141482352, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]];
var a=0;
var b=0;
var last_a=0;
var last_b=0;

function updateRange()
{
	if(last_a==a && last_b==b)
	{
		setTimeout(updateRange,500);
		return;
	}
 
        an=Math.floor(parseInt(a));
        bn=Math.floor(parseInt(b));
	var mn = 0;
	var mx = 0;
	for(i=an;i<bn;i++)
	{
		mx=Math.max(mx,cols[0][i+1]);
		mn=Math.min(mn,cols[1][i+1]);
	}
	r={}
	r.max=mx;
	r.min=mn;
	chart.axis.range(r);
	last_a=a;
	last_b=b;
	setTimeout(updateRange,500);
}
setTimeout(updateRange,500);

var chart = c3.generate({
bindto: '#plot_mapq_thresh',
data: {
  columns: [["Correct", 0.9673209126232938, 0.9673209126232938, 0.9673209126232938, 0.9673209126232938, 0.9672212812593405, 0.9672212812593405, 0.9672212812593405, 0.9672212812593405, 0.9672212812593405, 0.9672212812593405, 0.9671216498953871, 0.9670220185314337, 0.9669223871674804, 0.966822755803527, 0.9665238617116668, 0.9665238617116668, 0.9662249676198067, 0.9661253362558534, 0.9658264421639933, 0.9655275480721331, 0.9654279167081797, 0.9649297598884129, 0.9647304971605061, 0.9643319717046926, 0.9642323403407392, 0.963634552157019, 0.9633356580651589, 0.9626382385174853, 0.9623393444256252, 0.9615422935139982, 0.9611437680581847, 0.9579555644116768, 0.9565607253163296, 0.955066254857029, 0.9527747334861014, 0.9506824748430805, 0.9481916907442463, 0.944903855733785, 0.9408189698116968, 0.936634452525655, 0.9326491979675202, 0.9266713161303178, 0.9217893792966025, 0.9145162897280064, 0.9080402510710371, 0.9001693733187207, 0.8874165587326891, 0.8775530537013052, 0.8676895486699213, 0.8463684367838996, 0.8263425326292717, 0.7909733984258245, 0.7563016837700508, 0.7106705190794061, 0.6585633157317924, 0.5876257845969911, 0.500647603865697, 0.3985254558134901, 0.29520773139384276, 0.22705987844973596, 0.21908936933346618, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], ["Wrong", -0.9635570180216467, -0.9522653342167055, -0.9522653342167055, -0.9522653342167055, -0.9522653342167055, -0.9522653342167055, -0.9485014396150584, -0.9409736504117643, -0.9409736504117643, -0.9409736504117643, -0.9334458612084703, -0.9296819666068231, -0.9259180720051761, -0.9259180720051761, -0.9221541774035291, -0.9221541774035291, -0.9221541774035291, -0.9221541774035291, -0.918390282801882, -0.910862493598588, -0.9070985989969408, -0.8958069151919996, -0.8845152313870585, -0.8807513367854115, -0.8732235475821173, -0.854404074573882, -0.8393484961672938, -0.8130012339557644, -0.7715983933376468, -0.7301955527195291, -0.7038482905079998, -0.6737371336948232, -0.6360981876783527, -0.6097509254668233, -0.5834036632552939, -0.5269452442305881, -0.5156535604256469, -0.4893062982141175, -0.455431246799294, -0.41402840618117626, -0.3801533547663528, -0.3425144087498822, -0.31240325193670576, -0.28981988432682343, -0.27100041131858815, -0.24465314910705874, -0.2295975707004705, -0.22206978149717638, -0.2107780976922352, -0.18066694087905874, -0.16184746787082346, -0.15431967866752935, -0.10915294344776466, -0.09409736504117643, -0.0790417866345882, -0.06775010282964704, -0.06022231362635292, -0.04140284061811764, -0.03387505141482352, -0.03387505141482352, -0.03387505141482352, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]],
  type: "area"
},
grid: {
  y: {
    show: true
  }
},
axis: {
  x: {
    label: "Mapping Quality Threshold"
  },

  y: {
    label: { text: "Read Count", position: "outer-middle" },
    tick: {
       format: function(v) { if(v>=0){return Math.min(100,(Math.round(100000*v)/1000))+"%";}else{ return Math.min(100,(Math.round(100000*(-v/37.778210))/1000))+"%";} }
    }
  }
},
legend: {
	position: "right"
},
subchart: {
	show: true,
	onbrush: function (d) {
		a = d[0];
		b = d[1];
	}
}
});

</script>
<script src="http://teaser.cibiv.univie.ac.at/static/bootstrap/js/bootstrap.min.js"></script>
<script type="javascript">$('#content').scrollspy();</script>
<script src="http://teaser.cibiv.univie.ac.at/static/extensions/bootstrap-select.min.js"></script>

<script src="http://teaser.cibiv.univie.ac.at/static/DataTables-1.10.8/media/js/jquery.dataTables.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/extensions/dataTables.bootstrap.js"></script>
</body>
</html>
