
<!DOCTYPE html>
<meta charset="utf-8">
<html lang="en">
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Teaser - Rapid Personalized Benchmarks for NGS Read Mapping</title>
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/bootstrap/css/bootstrap.min.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/DataTables-1.10.8/media/css/jquery.dataTables.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/c3/c3.css">
    <link rel="stylesheet" href="http://teaser.cibiv.univie.ac.at/static/extensions/bootstrap-select.min.css">
    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
    <style>
body {
  padding-top: 60px;
  position: relative;
}

.sub-header {
  padding-bottom: 10px;
  border-bottom: 1px solid #eee;
}


.sidebar {
  display: none;
}

@media (min-width: 768px) {
  .sidebar {
    position: fixed;
    top: 51px;
    bottom: 0;
    left: 0;
    z-index: 1000;
    display: block;
    padding: 20px;
    overflow-x: hidden;
    overflow-y: auto; /* Scrollable contents if viewport is shorter than content. */
    background-color: #f5f5f5;
    border-right: 1px solid #eee;
  }
}

.nav-sidebar {
  margin-right: -21px;
  margin-bottom: 20px;
  margin-left: -20px;
}
.nav-sidebar > li > a {
  padding-right: 20px;
  padding-left: 20px;
}

.nav-sidebar > .active > a,
.nav-sidebar > .active > a:hover,
.nav-sidebar > .active > a:focus {
  color: #fff;
  background-color: #428bca;
}

a.separator
{
color:#777777;
}

a.separator:hover
{
color:#777777;
}

.main {
  padding: 20px;
}
@media (min-width: 768px) {
  .main {
    padding-right: 40px;
    padding-left: 40px;
  }
}
.main .page-header {
  margin-top: 0;
}

.placeholders {
  margin-bottom: 30px;
  text-align: center;
}
.placeholders h4 {
  margin-bottom: 0;
}
.placeholder {
  margin-bottom: 20px;
}
.placeholder img {
  display: inline-block;
  border-radius: 50%;
}

section {
    padding-top: 60px;
    margin-top: -60px;
}

.bar {
  fill: steelblue;
}

.bar:hover {
  fill: brown;
}

.axis {
  font: 10px sans-serif;
}

.axis path,
.axis line {
  fill: none;
  stroke: #000;
  shape-rendering: crispEdges;
}

.x.axis path {
  display: none;
}

.main-container
{
   margin: 0 auto;
   max-width: 1300px;
}


    </style>
  </head>

<body data-spy="scroll" data-target="#sidebar">
    <nav class="navbar navbar-inverse navbar-fixed-top">
      <div class="container-fluid">
        <div class="navbar-header">
          <a class="navbar-brand" href="/">Teaser</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav">
           <li><a href="start.html">Benchmark</a></li><li><a href="#" class="separator"><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></a></li><li><a href="e_coli_sim_clr.html">Sim CLR</a></li><li><a href="#" class="separator"><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></a></li><li class="dropdown"><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">GraphMap2<span class="caret"></span></a><ul class="dropdown-menu" role="menu"><li><a href="e_coli_sim_clr_lra.html">LRA (def)</a></li><li><a href="e_coli_sim_clr_ngmlr.html">NGMLR (def)</a></li><li><a href="e_coli_sim_clr_minimap2_01.html">Minimap2 (p1)</a></li><li><a href="e_coli_sim_clr_lra_01.html">LRA (p1)</a></li><li><a href="e_coli_sim_clr_winnowmap_01.html">Winnowmap2 (p1)</a></li><li><a href="e_coli_sim_clr_winnowmap.html">Winnowmap2 (def)</a></li><li><a href="e_coli_sim_clr_lordfast.html">lordFAST</a></li><li><a href="e_coli_sim_clr_minimap2.html">Minimap2 (def)</a></li><li><a href="e_coli_sim_clr_graphmap.html">GraphMap2</a></li><li><a href="e_coli_sim_clr_ngmlr_01.html">NGMLR (p1)</a></li></ul></li>
          </ul>
          <div class="nav navbar-nav navbar-right" style="padding:5px;">
           
          </div>
        </div>
      </div>
    </nav>

    <div class="container-fluid main-container">
      <div class="row">
        
        <div class="col-sm-3 col-md-2 sidebar">
          <div id="sidebar">
          <ul class="nav nav-sidebar">
            <li class="active"><a href="#section1">Mapping Quality Thresholds</a></li><li><a href="#section2">Basic Statistics</a></li><li><a href="#section3">Read Failure Statistics</a></li><li><a href="#section4">Advanced Statistics</a></li><li><a href="#section5">Timing</a></li><li><a href="#section6">Additional Information</a></li><li><a href="#section7">Errors and Warnings</a></li><li><a href="#section8">Subprocess Log</a></li>
          </ul>
          
          </div>
        </div>
        <div class="col-sm-9  col-sm-offset-3 col-md-offset-2  col-md-10 main" id="content">
            <section id="section1"><div class="panel panel-default"><div class="panel-heading">Mapping Quality Thresholds</div><div class="panel-body"><p>The plot below shows the percentages of correctly and wrongly mapped reads for all mapping quality thresholds for this mapper. The values at threshold 0 therefore correspond to the unfiltered results.</p><div id="plot_mapq_thresh"></div><div align="right" style="margin-top:5px;"><div class="dropdown">
  <button class="btn btn-default btn-xs dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
    <span class="glyphicon glyphicon-download-alt" aria-hidden="true"> Export
    <span class="caret"></span>
  </button>
  <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu1">
    <li><a href="javascript:exportSVG('plot_mapq_thresh');">Plot (.svg)</a></li>
    <li><a href="e_coli_sim_clr_graphmap_mapqs.csv">Raw Data  (.csv)</a></li>
  </ul>
</div></div></div></div></section><section id="section2"><div class="panel panel-default"><div class="panel-heading">Basic Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Correctly Mapped</th><td class="col-md-2">10027</td><td class="col-md-2">99.9%</td></tr><tr><th>Wrongly Mapped</th><td>5</td><td>0.05%</td></tr><tr><th>Not Mapped</th><td>5</td><td>0.05%</td></tr><tr><th>Total</th><td>10037</td><td>100.0%</td></tbody></table></div></div></div></section><section id="section3"><div class="panel panel-default"><div class="panel-heading">Read Failure Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Not mapped</th><td class="col-md-2">4</td><td class="col-md-2">0.04%</td></tr><tr><th>Missing in mapper output</th><td>1</td><td>0.01%</td></tr><tr><th>Mapped to wrong chromosome</th><td>0</td><td>0.0%</td></tr><tr><th>Mapped to wrong position</th><td>5</td><td>0.05%</td></tr><tr><th>Mapped to wrong strand</th><td>0</td><td>0.0%</td></tbody></table></div></div></div></section><section id="section4"><div class="panel panel-default"><div class="panel-heading">Advanced Statistics</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th>Missing in comparison alignment</th><td>1</td></tr><tr><th class="col-md-6">Secondary Alignments</th><td class="col-md-4">0</td></tr><tr><th><abbr title="Harmonic Mean of Precision and Recall">F-Measure</abbr></th><td>0.999600</td></tr><tr><th><abbr title="Correctly Mapped / (Correctly Mapped + Wrongly Mapped)">Precision</abbr></th><td>0.999500</td></tr><tr><th><abbr title="Correctly Mapped / (Correctly Mapped + Unmapped)">Recall</abbr></th><td>0.999600</td></tbody></table></div></div></div></section><section id="section5"><div class="panel panel-default"><div class="panel-heading">Timing</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Raw Mapping Time</th><td class="col-md-4">100.051s</td></tr><tr><th>Effective Mapping Time</th><td>99.656s</td></tr><tr><th>Effective Init Time</th><td>0.395s</td></tr><tr><th>Effective Time Measure</th><td>Wall clock</td></tr><tr><th>Mapping Time (Wall)</th><td>100.051s</td></tr><tr><th>Mapping Time (CPU)</th><td>746.546s</td></tr><tr><th>Mapping Time (CPU User)</th><td>648.302s</td></tr><tr><th>Mapping Time (CPU System)</th><td>98.244s</td></tr><tr><th>Init Time (Wall)</th><td>0.395s</td></tr><tr><th>Init Time (CPU)</th><td>0.408s</td></tr><tr><th>Init Time (CPU User)</th><td>0.229s</td></tr><tr><th>Init Time (CPU System)</th><td>0.179s</td></tr></tbody></table></div></div></div></section><section id="section6"><div class="panel panel-default"><div class="panel-heading">Additional Information</div><div class="panel-body"><div class="table-responsive"><table class="table table-striped"><tbody><tr><th class="col-md-6">Mapper Memory Usage</th><td class="col-md-4">631 MB</td></tr><tr><th>Total Test Runtime (Wall)</th><td>105.910s</td></tr><tr><th>Mapper Command Line:</th><td>&nbsp;</td></tr><tr><td colspan=2>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads.fastq -o out_graphmap.sam</td></tr></tbody></table></div></div></div></section><section id="section7"><div class="panel panel-default"><div class="panel-heading">Errors and Warnings</div><div class="panel-body"><i>No problems were encountered.</i></div></div></section><section id="section8"><div class="panel panel-default"><div class="panel-heading">Subprocess Log</div><div class="panel-body"><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -I /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -I /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse49" class="collapsed">stderr</a></h4></div><div id="collapse49" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse50" class="collapsed">stdout</a></h4></div><div id="collapse50" class="panel-collapse collapse"><div class="panel-body"><pre>ERROR: Unknown parameter value: '/home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta'. Exiting.<br>
<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>0.00291514396667</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_clr</small></td></tr></tbody></table></div></div></div><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads_base.fastq -o out_graphmap.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads_base.fastq -o out_graphmap.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>378972000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse51" class="collapsed">stderr</a></h4></div><div id="collapse51" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse52" class="collapsed">stdout</a></h4></div><div id="collapse52" class="panel-collapse collapse"><div class="panel-body"><pre>[13:22:31 BuildIndexes] Loading reference sequences.<br>
[13:22:31 SetupIndex_] Loading index from file: '/home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx'.<br>
[13:22:32 Index] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
[13:22:32 Run] Hits will be thresholded at the percentil value (percentil: 99.000000%, frequency: 6).<br>
[13:22:32 Run] Minimizers will be used. Minimizer window length: 5<br>
[13:22:32 Run] Automatically setting the maximum allowed number of regions: max. 500, attempt to reduce after 0<br>
[13:22:32 Run] Reference genome is assumed to be linear.<br>
[13:22:32 Run] Only one alignment will be reported per mapped read.<br>
[13:22:32 ProcessReads] Reads will be loaded in batches of up to 1024 MB in size.<br>
[13:22:32 ProcessReads] Batch of 1 reads (0 MiB) loaded in 0.00 sec. (94083241794752 bases)<br>
[13:22:32 ProcessReads] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
[13:22:32 ProcessReads] Using 8 threads.[13:22:32 ProcessReads] [CPU time: 0.00 sec, RSS: 269 MB] Read: 1/1 (100.00%) [m: 0, u: 1]                                                                         <br>
<br>
[13:22:32 ProcessReads] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
<br>
[13:22:32 ProcessReads] All reads processed in 0.00 sec (or 0.00 CPU min).<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>0.171119</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>0.412283182144</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>0.255622</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_clr</small></td></tr></tbody></table></div></div></div><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads_base.fastq -o out_graphmap.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads_base.fastq -o out_graphmap.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>378976000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse53" class="collapsed">stderr</a></h4></div><div id="collapse53" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse54" class="collapsed">stdout</a></h4></div><div id="collapse54" class="panel-collapse collapse"><div class="panel-body"><pre>[13:22:32 BuildIndexes] Loading reference sequences.<br>
[13:22:32 SetupIndex_] Loading index from file: '/home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx'.<br>
[13:22:33 Index] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
[13:22:33 Run] Hits will be thresholded at the percentil value (percentil: 99.000000%, frequency: 6).<br>
[13:22:33 Run] Minimizers will be used. Minimizer window length: 5<br>
[13:22:33 Run] Automatically setting the maximum allowed number of regions: max. 500, attempt to reduce after 0<br>
[13:22:33 Run] Reference genome is assumed to be linear.<br>
[13:22:33 Run] Only one alignment will be reported per mapped read.<br>
[13:22:33 ProcessReads] Reads will be loaded in batches of up to 1024 MB in size.<br>
[13:22:33 ProcessReads] Batch of 1 reads (0 MiB) loaded in 0.00 sec. (93969324362944 bases)<br>
[13:22:33 ProcessReads] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
[13:22:33 ProcessReads] Using 8 threads.[13:22:33 ProcessReads] [CPU time: 0.00 sec, RSS: 269 MB] Read: 1/1 (100.00%) [m: 0, u: 1]                                                                         <br>
<br>
[13:22:33 ProcessReads] Memory consumption: [currentRSS = 269 MB, peakRSS = 370 MB]<br>
<br>
[13:22:33 ProcessReads] All reads processed in 0.00 sec (or 0.00 CPU min).<br>
</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>0.178735</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>0.395313978195</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>0.229433</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_clr</small></td></tr></tbody></table></div></div></div><div class="panel panel-default"><div class="panel-heading"><h1 class="panel-title">/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads.fastq -o out_graphmap.sam</h1></div><div class="panel-body"><div class="table-responsive"><table class="table table-striped" style="margin-bottom;5px;"><tbody><tr><th class="col-md-2">command</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/software/graphmap2/bin/Linux-x64/graphmap2 align -a sg -t 8 -i /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx -r /home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta -d reads.fastq -o out_graphmap.sam</small></td></tr><tr><th class="col-md-2">memory</th><td class="col-md-8"><small>631764000</small></td></tr><tr><th class="col-md-2">return</th><td class="col-md-8"><small>0</small></td></tr><tr><th class="col-md-2">status</th><td class="col-md-8"><small>1</small></td></tr><tr><th class="col-md-2">stderr</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse55" class="collapsed">stderr</a></h4></div><div id="collapse55" class="panel-collapse collapse"><div class="panel-body"><pre>None</pre></div></div></div></small></td></tr><tr><th class="col-md-2">stdout</th><td class="col-md-8"><small><div class="panel panel-default" id="panel1"><div class="panel-heading"><h4 class="panel-title"><a href="javascript:void(0);" data-toggle="collapse" data-target="#collapse56" class="collapsed">stdout</a></h4></div><div id="collapse56" class="panel-collapse collapse"><div class="panel-body"><pre>[13:22:33 BuildIndexes] Loading reference sequences.<br>
[13:22:33 SetupIndex_] Loading index from file: '/home/raven/Desktop/Diploma/Teaser/references/E_coli.fasta.gmidx'.<br>
[13:22:34 Index] Memory consumption: [currentRSS = 269 MB, peakRSS = 369 MB]<br>
[13:22:34 Run] Hits will be thresholded at the percentil value (percentil: 99.000000%, frequency: 6).<br>
[13:22:34 Run] Minimizers will be used. Minimizer window length: 5<br>
[13:22:34 Run] Automatically setting the maximum allowed number of regions: max. 500, attempt to reduce after 0<br>
[13:22:34 Run] Reference genome is assumed to be linear.<br>
[13:22:34 Run] Only one alignment will be reported per mapped read.<br>
[13:22:34 ProcessReads] Reads will be loaded in batches of up to 1024 MB in size.<br>
[13:22:34 ProcessReads] Batch of 10037 reads (177 MiB) loaded in 0.68 sec. (140727564232352 bases)<br>
[13:22:34 ProcessReads] Memory consumption: [currentRSS = 438 MB, peakRSS = 438 MB]<br>
[13:22:34 ProcessReads] Using 8 threads.[13:22:34 ProcessReads] [CPU time: 0.68 sec, RSS: 438 MB] Read: 6/10037 (0.06%) [m: 0, u: 0], length = 28889, qname: S1_7                                          [13:22:35 ProcessReads] [CPU time: 2.93 sec, RSS: 475 MB] Read: 38/10037 (0.38%) [m: 31, u: 0], length = 9327, qname: S1_39                                        [13:22:35 ProcessReads] [CPU time: 3.31 sec, RSS: 482 MB] Read: 41/10037 (0.41%) [m: 34, u: 0], length = 4328, qname: S1_42                                        [13:22:35 ProcessReads] [CPU time: 3.47 sec, RSS: 485 MB] Read: 43/10037 (0.43%) [m: 36, u: 0], length = 2320, qname: S1_44                                        [13:22:35 ProcessReads] [CPU time: 3.54 sec, RSS: 485 MB] Read: 45/10037 (0.45%) [m: 38, u: 0], length = 10435, qname: S1_46                                       [13:22:35 ProcessReads] [CPU time: 4.11 sec, RSS: 488 MB] Read: 55/10037 (0.55%) [m: 48, u: 0], length = 17408, qname: S1_56                                       [13:22:35 ProcessReads] [CPU time: 5.19 sec, RSS: 495 MB] Read: 73/10037 (0.73%) [m: 66, u: 0], length = 6766, qname: S1_74                                        [13:22:35 ProcessReads] [CPU time: 5.47 sec, RSS: 495 MB] Read: 79/10037 (0.79%) [m: 72, u: 0], length = 10409, qname: S1_80                                       [13:22:35 ProcessReads] [CPU time: 5.93 sec, RSS: 501 MB] Read: 88/10037 (0.88%) [m: 81, u: 0], length = 5863, qname: S1_89                                        [13:22:35 ProcessReads] [CPU time: 6.18 sec, RSS: 501 MB] Read: 90/10037 (0.90%) [m: 83, u: 0], length = 15582, qname: S1_91                                       [13:22:35 ProcessReads] [CPU time: 6.99 sec, RSS: 509 MB] Read: 100/10037 (1.00%) [m: 93, u: 0], length = 8174, qname: S1_101                                      [13:22:35 ProcessReads] [CPU time: 7.28 sec, RSS: 511 MB] Read: 104/10037 (1.04%) [m: 97, u: 0], length = 10556, qname: S1_105                                     [13:22:35 ProcessReads] [CPU time: 7.81 sec, RSS: 511 MB] Read: 108/10037 (1.08%) [m: 101, u: 0], length = 8919, qname: S1_109                                     [13:22:35 ProcessReads] [CPU time: 8.39 sec, RSS: 511 MB] Read: 117/10037 (1.17%) [m: 110, u: 0], length = 7762, qname: S1_118                                     [13:22:35 ProcessReads] [CPU time: 8.67 sec, RSS: 511 MB] Read: 120/10037 (1.20%) [m: 113, u: 0], length = 4773, qname: S1_121                                     [13:22:35 ProcessReads] [CPU time: 8.85 sec, RSS: 511 MB] Read: 121/10037 (1.21%) [m: 114, u: 0], length = 1071, qname: S1_122                                     [13:22:35 ProcessReads] [CPU time: 8.88 sec, RSS: 512 MB] Read: 122/10037 (1.22%) [m: 115, u: 0], length = 9250, qname: S1_123                                     [13:22:35 ProcessReads] [CPU time: 9.34 sec, RSS: 514 MB] Read: 132/10037 (1.32%) [m: 125, u: 0], length = 6817, qname: S1_133                                     [13:22:35 ProcessReads] [CPU time: 9.60 sec, RSS: 515 MB] Read: 135/10037 (1.35%) [m: 128, u: 0], length = 2309, qname: S1_136                                     [13:22:36 ProcessReads] [CPU time: 9.66 sec, RSS: 515 MB] Read: 137/10037 (1.36%) [m: 130, u: 0], length = 3702, qname: S1_138                                     [13:22:36 ProcessReads] [CPU time: 9.78 sec, RSS: 515 MB] Read: 139/10037 (1.38%) [m: 132, u: 0], length = 6542, qname: S1_140                                     [13:22:36 ProcessReads] [CPU time: 10.05 sec, RSS: 517 MB] Read: 141/10037 (1.40%) [m: 134, u: 0], length = 3751, qname: S1_142                                    [13:22:36 ProcessReads] [CPU time: 10.18 sec, RSS: 517 MB] Read: 143/10037 (1.42%) [m: 136, u: 0], length = 6452, qname: S1_144                                    [13:22:36 ProcessReads] [CPU time: 10.49 sec, RSS: 517 MB] Read: 145/10037 (1.44%) [m: 138, u: 0], length = 5337, qname: S1_146                                    [13:22:36 ProcessReads] [CPU time: 10.66 sec, RSS: 520 MB] Read: 147/10037 (1.46%) [m: 140, u: 0], length = 6116, qname: S1_148                                    [13:22:36 ProcessReads] [CPU time: 10.97 sec, RSS: 520 MB] Read: 149/10037 (1.48%) [m: 142, u: 0], length = 5544, qname: S1_150                                    [13:22:36 ProcessReads] [CPU time: 11.32 sec, RSS: 520 MB] Read: 153/10037 (1.52%) [m: 146, u: 0], length = 6758, qname: S1_154                                    [13:22:36 ProcessReads] [CPU time: 11.66 sec, RSS: 520 MB] Read: 155/10037 (1.54%) [m: 148, u: 0], length = 19992, qname: S1_156                                   [13:22:36 ProcessReads] [CPU time: 13.31 sec, RSS: 522 MB] Read: 172/10037 (1.71%) [m: 165, u: 0], length = 18736, qname: S1_173                                   [13:22:36 ProcessReads] [CPU time: 14.62 sec, RSS: 530 MB] Read: 184/10037 (1.83%) [m: 177, u: 0], length = 18995, qname: S1_185                                   [13:22:36 ProcessReads] [CPU time: 15.79 sec, RSS: 531 MB] Read: 201/10037 (2.00%) [m: 194, u: 0], length = 4084, qname: S1_202                                    [13:22:36 ProcessReads] [CPU time: 15.94 sec, RSS: 531 MB] Read: 204/10037 (2.03%) [m: 197, u: 0], length = 3107, qname: S1_205                                    [13:22:36 ProcessReads] [CPU time: 16.04 sec, RSS: 531 MB] Read: 206/10037 (2.05%) [m: 199, u: 0], length = 6373, qname: S1_207                                    [13:22:36 ProcessReads] [CPU time: 16.29 sec, RSS: 531 MB] Read: 208/10037 (2.07%) [m: 201, u: 0], length = 1557, qname: S1_209                                    [13:22:36 ProcessReads] [CPU time: 16.35 sec, RSS: 531 MB] Read: 209/10037 (2.08%) [m: 202, u: 0], length = 11770, qname: S1_210                                   [13:22:36 ProcessReads] [CPU time: 16.89 sec, RSS: 531 MB] Read: 218/10037 (2.17%) [m: 211, u: 0], length = 6961, qname: S1_219                                    [13:22:36 ProcessReads] [CPU time: 17.13 sec, RSS: 536 MB] Read: 222/10037 (2.21%) [m: 215, u: 0], length = 6409, qname: S1_223                                    [13:22:37 ProcessReads] [CPU time: 17.34 sec, RSS: 536 MB] Read: 225/10037 (2.24%) [m: 218, u: 0], length = 13523, qname: S1_226                                   [13:22:37 ProcessReads] [CPU time: 18.16 sec, RSS: 536 MB] Read: 230/10037 (2.29%) [m: 223, u: 0], length = 6889, qname: S1_231                                    [13:22:37 ProcessReads] [CPU time: 18.40 sec, RSS: 536 MB] Read: 234/10037 (2.33%) [m: 227, u: 0], length = 7268, qname: S1_235                                    [13:22:37 ProcessReads] [CPU time: 18.67 sec, RSS: 537 MB] Read: 238/10037 (2.37%) [m: 231, u: 0], length = 2769, qname: S1_239                                    [13:22:37 ProcessReads] [CPU time: 18.74 sec, RSS: 537 MB] Read: 239/10037 (2.38%) [m: 232, u: 0], length = 14292, qname: S1_240                                   [13:22:37 ProcessReads] [CPU time: 19.39 sec, RSS: 537 MB] Read: 257/10037 (2.56%) [m: 250, u: 1], length = 2583, qname: S1_258                                    [13:22:37 ProcessReads] [CPU time: 19.48 sec, RSS: 537 MB] Read: 259/10037 (2.58%) [m: 251, u: 1], length = 1751, qname: S1_260                                    [13:22:37 ProcessReads] [CPU time: 19.53 sec, RSS: 537 MB] Read: 261/10037 (2.60%) [m: 253, u: 1], length = 7757, qname: S1_262                                    [13:22:37 ProcessReads] [CPU time: 19.84 sec, RSS: 537 MB] Read: 266/10037 (2.65%) [m: 258, u: 1], length = 9707, qname: S1_267                                    [13:22:37 ProcessReads] [CPU time: 20.24 sec, RSS: 538 MB] Read: 268/10037 (2.67%) [m: 260, u: 1], length = 4694, qname: S1_269                                    [13:22:37 ProcessReads] [CPU time: 20.39 sec, RSS: 538 MB] Read: 270/10037 (2.69%) [m: 262, u: 1], length = 8357, qname: S1_271                                    [13:22:37 ProcessReads] [CPU time: 20.69 sec, RSS: 538 MB] Read: 276/10037 (2.75%) [m: 268, u: 1], length = 5481, qname: S1_277                                    [13:22:37 ProcessReads] [CPU time: 20.85 sec, RSS: 538 MB] Read: 281/10037 (2.80%) [m: 273, u: 1], length = 9435, qname: S1_282                                    [13:22:37 ProcessReads] [CPU time: 21.23 sec, RSS: 538 MB] Read: 287/10037 (2.86%) [m: 279, u: 1], length = 34327, qname: S1_288                                   [13:22:38 ProcessReads] [CPU time: 24.83 sec, RSS: 544 MB] Read: 340/10037 (3.39%) [m: 332, u: 1], length = 3354, qname: S1_341                                    [13:22:38 ProcessReads] [CPU time: 24.96 sec, RSS: 544 MB] Read: 345/10037 (3.44%) [m: 337, u: 1], length = 7068, qname: S1_346                                    [13:22:38 ProcessReads] [CPU time: 25.24 sec, RSS: 544 MB] Read: 350/10037 (3.49%) [m: 342, u: 1], length = 1226, qname: S1_351                                    [13:22:38 ProcessReads] [CPU time: 25.28 sec, RSS: 544 MB] Read: 351/10037 (3.50%) [m: 343, u: 1], length = 8101, qname: S1_352                                    [13:22:38 ProcessReads</pre></div></div></div></small></td></tr><tr><th class="col-md-2">systime</th><td class="col-md-8"><small>98.244191</small></td></tr><tr><th class="col-md-2">time</th><td class="col-md-8"><small>100.050857067</small></td></tr><tr><th class="col-md-2">usrtime</th><td class="col-md-8"><small>648.301989</small></td></tr><tr><th class="col-md-2">working_directory</th><td class="col-md-8"><small>/home/raven/Desktop/Diploma/Teaser/tests_generated/e_coli_sim_clr</small></td></tr></tbody></table></div></div></div></div></div></section><p></p><div style="margin:300px; height:300px;">&nbsp;</div>
        </div>
      </div>
    </div>


<script src="http://teaser.cibiv.univie.ac.at/static/js/d3.v3.min.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/c3/c3.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/js/jquery.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/js/svgexport.js"></script>
<script>

var cols=[["Correct", 0.9990036863604663, 0.9990036863604663, 0.9990036863604663, 0.9988044236325595, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], ["Wrong", -0.8325030719670553, -0.8325030719670553, -0.8325030719670553, -0.3330012287868221, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]];
var a=0;
var b=0;
var last_a=0;
var last_b=0;

function updateRange()
{
	if(last_a==a && last_b==b)
	{
		setTimeout(updateRange,500);
		return;
	}
 
        an=Math.floor(parseInt(a));
        bn=Math.floor(parseInt(b));
	var mn = 0;
	var mx = 0;
	for(i=an;i<bn;i++)
	{
		mx=Math.max(mx,cols[0][i+1]);
		mn=Math.min(mn,cols[1][i+1]);
	}
	r={}
	r.max=mx;
	r.min=mn;
	chart.axis.range(r);
	last_a=a;
	last_b=b;
	setTimeout(updateRange,500);
}
setTimeout(updateRange,500);

var chart = c3.generate({
bindto: '#plot_mapq_thresh',
data: {
  columns: [["Correct", 0.9990036863604663, 0.9990036863604663, 0.9990036863604663, 0.9988044236325595, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.9983062668127927, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], ["Wrong", -0.8325030719670553, -0.8325030719670553, -0.8325030719670553, -0.3330012287868221, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]],
  type: "area"
},
grid: {
  y: {
    show: true
  }
},
axis: {
  x: {
    label: "Mapping Quality Threshold"
  },

  y: {
    label: { text: "Read Count", position: "outer-middle" },
    tick: {
       format: function(v) { if(v>=0){return Math.min(100,(Math.round(100000*v)/1000))+"%";}else{ return Math.min(100,(Math.round(100000*(-v/1671.166667))/1000))+"%";} }
    }
  }
},
legend: {
	position: "right"
},
subchart: {
	show: true,
	onbrush: function (d) {
		a = d[0];
		b = d[1];
	}
}
});

</script>
<script src="http://teaser.cibiv.univie.ac.at/static/bootstrap/js/bootstrap.min.js"></script>
<script type="javascript">$('#content').scrollspy();</script>
<script src="http://teaser.cibiv.univie.ac.at/static/extensions/bootstrap-select.min.js"></script>

<script src="http://teaser.cibiv.univie.ac.at/static/DataTables-1.10.8/media/js/jquery.dataTables.js"></script>
<script src="http://teaser.cibiv.univie.ac.at/static/extensions/dataTables.bootstrap.js"></script>
</body>
</html>
